### PR TITLE
Disaster recovery cloud deploy 2.0

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-9796

This PR includes Cloud 2.0 updates for pages under the Backup and Restore section of the Cloud Deployment part of the nav.

### CODE REVIEW

File: src/current/cockroachcloud/backup-and-restore-overview.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ CockroachDB Cloud clusters can run two types of backups:
...
 
 ### Customer-owned backups
 
-You can take manual backups and store them in your [cloud storage buckets](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/use-cloud-storage) using the [`BACKUP`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup) statement. Customer-owned backups are supported in CockroachDB {{ site.data.products.serverless }} and CockroachDB {{ site.data.products.dedicated }}. Refer to the [Backup and restore support](#backup-and-restore-support) table for a list of the types of manual backups that are available.
+You can take manual backups and store them in your [cloud storage buckets](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/use-cloud-storage) using the [`BACKUP`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup) statement. Customer-owned backups are supported in all CockroachDB {{ site.data.products.cloud }}. Refer to the [Backup and restore support](#backup-and-restore-support) table for a list of the types of manual backups that are available.
```
Comment (florence-crl) [inline src/current/cockroachcloud/backup-and-restore-overview.md:26]:
```suggestion
You can take manual backups and store them in your [cloud storage buckets](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/use-cloud-storage) using the [`BACKUP`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup) statement. Customer-owned backups are supported in all CockroachDB {{ site.data.products.cloud }} products. Refer to the [Backup and restore support](#backup-and-restore-support) table for a list of the types of manual backups that are available.
```

File: src/current/v23.1/backup-architecture.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ During a [restore]({% link {{ page.version.version }}/restore.md %}) job, the jo
 
 #### Job coordination on Serverless clusters
```
Comment (florence-crl) [inline src/current/v23.1/backup-architecture.md:121]: Should `Serverless` be replaced in this header?

File: src/current/v23.2/backup-architecture.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ During a [restore]({% link {{ page.version.version }}/restore.md %}) job, the jo
 
 #### Job coordination on Serverless clusters
```
Comment (florence-crl) [inline src/current/v23.2/backup-architecture.md:121]: Should `Serverless` be replaced in this header?

File: src/current/v23.1/take-and-restore-locality-aware-backups.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ For a technical overview of how a locality-aware backup works, refer to [Job coo
 Locality-aware backups are available in **CockroachDB {{ site.data.products.dedicated }}**, **CockroachDB {{ site.data.products.serverless }}**, and **CockroachDB {{ site.data.products.core }}** clusters when you are running [customer-owned backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}#cockroachdb-backup-types). For a full list of features, see [Backup and restore product support]({% link {{ page.version.version }}/backup-and-restore-overview.md %}#backup-and-restore-product-support).
```
Comment (florence-crl) [inline src/current/v23.1/take-and-restore-locality-aware-backups.md:28]: Should these dedicated and serverless variables be updated?

File: src/current/v23.2/take-and-restore-locality-aware-backups.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ For a technical overview of how a locality-aware backup works, refer to [Job coo
 Locality-aware backups are available in **CockroachDB {{ site.data.products.dedicated }}**, **CockroachDB {{ site.data.products.serverless }}**, and **CockroachDB {{ site.data.products.core }}** clusters when you are running [customer-owned backups](https://www.cockroachlabs.com/docs/cockroachcloud/take-and-restore-customer-owned-backups). For a full list of features on CockroachDB Cloud, refer to [Backup and Restore Overview](https://www.cockroachlabs.com/docs/cockroachcloud/backup-and-restore-overview).
```
Comment (florence-crl) [inline src/current/v23.2/take-and-restore-locality-aware-backups.md:28]: Should these dedicated and serverless variables be updated?

**Review by florence-crl — APPROVED** [review]
LGTM pending questions being addressed.

The markdown code looks correct, however these rendered preview pages do not show the include files with the updated product names: 

src/current/_includes/v23.1/backups/locality-aware-multi-tenant.md:
[src/current/v23.1/take-and-restore-locality-aware-backups.md](https://deploy-preview-18358--cockroachdb-docs.netlify.app/docs/v23.1/take-and-restore-locality-aware-backups.html)
[src/current/v23.1/backup-architecture.md](https://deploy-preview-18358--cockroachdb-docs.netlify.app/docs/v23.1/backup-architecture.html)
src/current/_includes/v23.2/backups/locality-aware-multi-tenant.md:
[src/current/v23.2/take-and-restore-locality-aware-backups.md](https://deploy-preview-18358--cockroachdb-docs.netlify.app/docs/v23.2/take-and-restore-locality-aware-backups.html)
[src/current/v23.2/backup-architecture.md](https://deploy-preview-18358--cockroachdb-docs.netlify.app/docs/v23.2/backup-architecture.html)

File: src/current/v23.2/take-and-restore-locality-aware-backups.md (hunk 5 above)
Comment (kathancox) [reply src/current/v23.2/take-and-restore-locality-aware-backups.md:28]: Yes! But I'll do it in my next PR as it is part of self-hosted nav section.

File: src/current/v23.1/backup-architecture.md (hunk 2 above)
Comment (kathancox) [reply src/current/v23.1/backup-architecture.md]: Yes, Done!

**Pull request merged**